tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -14,3 +14,19 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<button
+  class="mat-focus-indicator mat-icon-button mat-button-base"
+  mat-icon-button
+>
+  <span class="mat-button-wrapper">
+    <mat-icon
+      class="mat-icon mat-icon-no-color notranslate"
```

Comment:
These class names feel very odd to me. If you look at the Angular doc <https://material.angular.io/components/button/examples>, you would not need those classes.

![image](https://user-images.githubusercontent.com/2547313/109573190-648aad80-7aa3-11eb-8444-4fac768c5440.png)

Reply:
Classes of which element (<button> or <mat-icon>)?
I'm following other icons' classes:
![Screen Shot 2021-03-01 at 3 55 43 PM](https://user-images.githubusercontent.com/1131010/109575422-9a7d6100-7aa6-11eb-898e-02203d710917.png)



Without the classes in <button> the ui breaks
![Screen Shot 2021-03-01 at 3 53 11 PM](https://user-images.githubusercontent.com/1131010/109575299-62761e00-7aa6-11eb-8413-cb47313d43ff.png)

Without the classes in mat-icon
![Screen Shot 2021-03-01 at 3 55 08 PM](https://user-images.githubusercontent.com/1131010/109575374-86d1fa80-7aa6-11eb-9591-da0643ad873a.png)

Reply:
Do you have a branch that I can play with? This is surprisingly; it should not happen this way.

Reply:
Thanks for the feedback and pr. Used MatButtonModule instead.